pkg/api/types.go

```diff
@@ -388,8 +391,21 @@ const (
 
 	// MountOptionAnnotation defines mount option annotation used in PVs
 	MountOptionAnnotation = "volume.beta.kubernetes.io/mount-options"
+
+	// StorageTopologyConstraint defines topology constraints of a PV
+	// Value is type []TopologyConstraint
+	AlphaStorageTopologyConstraintsAnnotation = "volume.alpha.kubernetes.io/topology-constraints"
 )
 
+// TopologyConstraint contains label information that the resource is constrained to
+// A label key can have multiple values
+type TopologyConstraint struct {
+	// Key of the label
+	Key string
```

Comment:
This is the internal API types.go, which does not have json.  The v1 types.go includes the json.